app/letters/utils.py

```diff
@@ -117,6 +123,46 @@ def delete_pdf_from_letters_scan_bucket(filename):
     current_app.logger.info("Deleted letter PDF: {}/{}".format(bucket_name, filename))
 
 
+def move_failed_pdf(filename, scan_error_type):
```

Comment:
Code in the function should be refactored so we are not duplicating code, could even update the existing `move_scanned_pdf_to_test_or_live_pdf_bucket` function to pass an additional argument for the `scan_error_type`